x-pack/legacy/plugins/maps/common/constants.js

```diff
@@ -37,6 +37,8 @@ export const ES_SIZE_LIMIT = 10000;
 export const FEATURE_ID_PROPERTY_NAME = '__kbn__feature_id__';
 export const FEATURE_VISIBLE_PROPERTY_NAME = '__kbn__isvisible__';
 
+export const MB_SOURCE_ID_LAYER_ID_PREFIX_DELIMITER = '_';
```

Comment:
Want to use a different delimiter since `_` can be common. Maybe something like `__` (2 `_`)